jekyll/_cci2/reusing-config.md

```diff
@@ -193,9 +195,7 @@ jobs:
       - run: echo outside the executor
 ```
 
-**Note:** Reusable `executor` declarations are available in configuration version 2.1 and later.
-
-## **`executors`**
+### The `executors` Key
```

Comment:
I believe this change is causing our html proofer to break as `jekyll/_site/docs/2.0/orbs-faq/index.html` links to `/docs/2.0/reusing-config/#executors`

Reply:
Ah yes thank you!